source/_patterns/04-components/back-to-top/back-to-top.twig

```diff
@@ -0,0 +1,5 @@
+<a href="#{{ top_element }}" class="back-to-top" title="{{ text }}" aria-hidden="true">
+  <svg class="back-to-top__icon" role="img" aria-label="{{ text }}">
+    <use xlink:href="{{ image_path }}/sprite.artifact.svg#arrow"></use>
+  </svg>
```

Comment:
What’s the rationale behind baking this into the back-to-top component rather than using the icon component?

example:
```
{% include '@components/icon/icon.twig' with {
  'modifier_classes': 'icon--back-to-top',
  'is_hidden': true,
  'icon_name': 'arrow'
} %}
```
We would also need to add `aria-label` to the anchor tag if we went with this approach.

Reply:
I did consider using the icon component here (and in fact started to write it that way) but ultimately decided not to because I didn't want the associated styling from the .icon class and I wanted slightly different markup. If the icon component had been larger/more complex, I would have gone with the variant class approach, but here, setting new styling to override the previous styling felt like overkill.